backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -689,13 +695,10 @@ interface CreationMap {
   creations: Decimal[]
 }
 
-export const getUserCreation = async (id: number, includePending = true): Promise<Decimal[]> => {
-  logger.trace('getUserCreation', id, includePending)
-  const creations = await getUserCreations([id], includePending)
-  return creations[0] ? creations[0].creations : FULL_CREATION_AVAILABLE
-}
-
-async function getUserCreations(ids: number[], includePending = true): Promise<CreationMap[]> {
+export async function getUserCreations(
```

Comment:
Why did you change this. We use the `=>` notations for functions

Reply:
Move this to `utils` as well?

Reply:
Yes, was the refactoring function of my ide will, I will change this!